src/Resources/skeleton/twig/Extension.tpl.php

```diff
@@ -12,19 +12,14 @@ public function getFilters(): array
             // If your filter generates SAFE HTML, you should add a third
             // parameter: ['is_safe' => ['html']]
             // Reference: https://twig.symfony.com/doc/3.x/advanced.html#automatic-escaping
-            new TwigFilter('filter_name', [$this, 'doSomething']),
+            new TwigFilter('filter_name', [<?= $runtime_class_name ?>, 'doSomething']),
```

Comment:
Per my comment earlier, this would now be:

```suggestion
            new TwigFilter('filter_name', [<?= $runtime_class_name ?>::class, 'doSomething']),
```